apps/vscode/src/providers/views/Account/AccountWebview.tsx

```
@@ -16,10 +16,10 @@
 
 import React, { useState, useCallback, useRef, useEffect } from 'react';
 
-import AccountView from 'shared/modules/account/AccountView';
-import { CheckoutModal } from 'shared/modules/checkout';
-import type { ApiKeyRecord, OrgDetail, MemberRecord, TeamRecord, TeamDetail, AccountSection, ProfileUpdate } from 'shell/src/components/BoxIcon';
-import type { CheckoutPlan, PromoRedemption, PromoValidation } from 'shared/modules/checkout';
+import { AccountView } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. No 'shell' barrel here: use relative imports (shell package) or deep 'shared/<group>' specs (vscode).

(no-restricted-imports)